packages/format/package.json

```diff
@@ -28,11 +30,11 @@
     "dist"
   ],
   "scripts": {
-    "prebuild": "pnpm run clean:build",
     "build": "tsup",
     "clean": "pnpm run clean:node && pnpm run clean:build",
-    "clean:node": "rm -rf node_modules",
     "clean:build": "rm -rf dist",
+    "clean:node": "rm -rf node_modules",
+    "prebuild": "pnpm run clean:build",
```

Comment:
**praise:** thanks for sorting them alphabetically!

Reply:
appreciated you noted, [finepack](https://github.com/Kikobeats/finepack) did it 🎉